pkgs/by-name/wi/wivrn/package.nix

```
@@ -0,0 +1,120 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchgit
+, avahi
+, boost
+, cmake
+, cudaPackages
+, eigen
+, ffmpeg
+, freetype
+, git
+, glm
+, glslang
+, harfbuzz
+, libdrm
+, libva
+, libpulseaudio
+, monado
+, nlohmann_json
+, onnxruntime
+, openxr-loader
+, pkg-config
+, python3
+, shaderc
+, spdlog
+, systemd
+, udev
+, vulkan-headers
+, vulkan-loader
+, vulkan-tools
+, x264
+, xorg
```

```suggestion
, libX11
, libXrandr
```

attrs under xorg are passed in by `callPackage`